src/scipp/core/variable.py

```diff
@@ -70,23 +92,32 @@ def zeros(*,
           with_variances: bool = False) -> Variable:
     """Constructs a :class:`Variable` with default initialized values with
     given dimension labels and shape.
+
     The dims and shape can also be specified using a sizes dict.
     Optionally can add default initialized variances.
     Only keyword arguments accepted.
 
-
-    :param dims: Optional (if sizes is specified), dimension labels.
-    :param shape: Optional (if sizes is specified), dimension sizes.
-    :param sizes: Optional, dimension label to size map.
-    :param unit: Optional, unit of contents. Default=dimensionless
-    :param dtype: Optional, type of underlying data. Default=float64
-    :param with_variances: Optional, boolean flag, if True includes variances
-      initialized to the default value for dtype.
-      For example for a float type values and variances would all be
-      initialized to 0.0. Default=False
-
-    :seealso: :py:func:`scipp.ones` :py:func:`scipp.empty`
-              :py:func:`scipp.scalar` :py:func:`scipp.array`
+    Parameters
+    ----------
+    dims:
+        Optional (if sizes is specified), dimension labels.
+    shape:
+        Optional (if sizes is specified), dimension sizes.
+    sizes:
+        Optional, dimension label to size map.
+    unit:
+        Optional, unit of contents. Default=dimensionless
```

Comment:
Isn't it the default unit? If we support zeros for non-numeric types that would not be the same?

Reply:
Removed the `Default=` parts from the docstrings. sphinx takes care of this.